### PR TITLE
Grey/fix elixir version bump

### PR DESCRIPTION
**Pull request by rgkeith3**
use operator branch: [update-dockerfile-image](https://gitlab.podium.com/engineering/voice/operator/-/tree/update-dockerfile-image)

replace ex_twilio in mix.exs with: 
{:ex_twilio,
       git: "https://github.com/rgkeith3/ex_twilio",
       ref: "114d197a78b489cb5e869f02667c4a477406326d"},

deploy operator
ensure everything works

### CODE REVIEW

**Review by jakerobers — APPROVED** [review]

File: mix.exs [hunk 1]
```diff
@@ -2,7 +2,7 @@ defmodule ExTwilio.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/danielberkompas/ex_twilio"
-  @version "0.9.1"
+  @version "0.9.2"
```
Comment (brettbeatty) [inline mix.exs:5]: Why do we need to bump the version? As far as I know `.tool-versions` and `mix.lock` are both project-level files and have no bearing when the app is included in another project

**Review by brettbeatty — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 1 above)
Comment (rgkeith3) [reply mix.exs:5]: ahh i guess you're right. although it does make me wonder if we should not just update the mix.lock, but the required versions of those packages in the mix file?

Comment (rgkeith3) [reply mix.exs:5]: I guess I'm also not sure, does updating the packages in the mix lock actually mean when operator pulls in ex_twilio, without a version change, and without any actual version updates in the ex_twilio/mix.exs, will these updates in the mix.lock make their way into operator?

Comment (brettbeatty) [reply mix.exs:5]: So some of these deps upgrades are required for the latest erlang? I'm not sure the best way to notate that. We could update the requirements in mix.exs and have dependent apps upgrade to the new patch version when the upgrade their erlang version. Or we could just document that the transitive dependencies need upgraded.

But no, as far as I'm aware the `mix.lock` does not get inherited by dependent apps (and I believe isn't even included in a package).

**Pull request closed without merging**